src/modules/app-db/repositories/nft.repository.ts

```diff
@@ -310,6 +310,17 @@ export class NftRepository {
     }
   }
 
+  async createWallet(walletAddress: string) {

```

Comment:
because the function creates wallet only when it does not exist I suggest changing the name to ensureWallet or createWalletIfNotExist etc...